tests/misc_env/setup_non_root_user.py

```diff
@@ -0,0 +1,20 @@
+from cli.exceptions import OperationFailedError
```

Comment:
instead of separate test case, add this one as method and provide option in bootstrap to setup non--root user.